tests/sentry/api/endpoints/test_organization_prevent_github_repos.py

```diff
@@ -0,0 +1,166 @@
+from unittest.mock import Mock, patch
+
+from sentry.testutils.cases import APITestCase
+
+
+class OrganizationPreventGitHubReposTest(APITestCase):
```

Comment:
nit: APITestCase gives you a bunch of helpers to not have to manually request the URLs. see https://github.com/getsentry/sentry/blob/dbd19309f015664769d092d0a261700fcc5564cd/tests/sentry/api/endpoints/test_custom_rules.py#L17 for an example. you set the `endpoint` and `method` on the test class and you can you `self.get_response`